examples/basic/index.html

```diff
@@ -153,15 +151,23 @@
 }
 
 function onResize() {
-  console.log('Resizing to %s x %s.', window.innerWidth, window.innerHeight);
-  effect.setSize(window.innerWidth, window.innerHeight);
-  camera.aspect = window.innerWidth / window.innerHeight;
-  camera.updateProjectionMatrix();
+  // The delay ensures the browser has a chance to layout
+  // the page and update the clientWidth/clientHeight.
+  if(!this.resizeDelay) {
+    this.resizeDelay = setTimeout(() => {
+      this.resizeDelay = null;
+      console.log('Resizing to %s x %s.', canvas.clientWidth, canvas.clientHeight);
+      effect.setSize(canvas.clientWidth, canvas.clientHeight, false);
+      camera.aspect = canvas.clientWidth / canvas.clientHeight;
+      camera.updateProjectionMatrix();
+    }, 250);
```

Comment:
Is this an iOS issue with the debounced resizing? Either way, should have a comment about it, it's not obvious to me from the code.